src/System.Text.Json/tests/Serialization/DictionaryTests.cs

```diff
@@ -124,20 +124,41 @@ public static void InvalidJsonForValueShouldFail(Type type, string json)
             Assert.Throws<JsonException>(() => JsonSerializer.Parse(json, type));
         }
 
+        [InlineData(typeof(int[]), @"""test""")]
+        [InlineData(typeof(int[]), @"1")]
+        [InlineData(typeof(int[]), @"false")]
+        [InlineData(typeof(int[]), @"{}")]
+        [InlineData(typeof(int[]), @"[""test""")]
+        [InlineData(typeof(int[]), @"[true]")]
+        // [InlineData(typeof(int[]), @"[{}]")] Fix is in progress
```

Comment:
Is there an issue number?  It'd be good to include e.g. `TODO #12345: Uncomment when fixed`